event/src/Event.cxx

```diff
@@ -10,7 +10,7 @@
 Event::Event() {
     
     // Create the tree
-    tree_ = new TTree("HPS_Event", "HPS event tree");
+    //tree_ = new TTree("HPS_Event", "HPS event tree");
```

Comment:
I agree with not creating the TTree inside the event. This _does_ change behavior of the Event class though.